third_party/libxml/BUILD.gn

```diff
@@ -6,9 +6,10 @@
 # headers.  These were generated by running the configure script offline.
 if (is_starboard) {
   os_include = "starboard"
-} else if (is_linux || is_android || is_nacl || is_fuchsia) {
+} else
+if (is_linux || is_chromeos || is_android || is_nacl || is_fuchsia) {
```

Comment:
Tiny nit, but this looks odd - did upstream break this in two lines ?

Reply:
I did this to clean up the diff:
```
 # Define an "os_include" variable that points at the OS-specific generated
 # headers.  These were generated by running the configure script offline.
+if (is_starboard) {
+  os_include = "starboard"
+} else
 if (is_linux || is_chromeos || is_android || is_nacl || is_fuchsia) {
   os_include = "linux"
```
Modified lines cause more merge conflicts and make it harder to avoid [evil merges](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefevilmergeaevilmerge).